script.js

```diff
@@ -0,0 +1,165 @@
+const toDoForm = document.querySelector("form");
+const toDoInput = toDoForm.querySelector(".input-text");
+const toDoButton = toDoForm.querySelector(".input-button");
+
+const TODO_DB = "todo";
+let toDoList = [];
+
+// localStorage 모델: 웹이 로드 될 때 기존 투두리스트를 가져오기 위함
+const localStorageModel = {
+    // localStorage에 저장
+    save: () => {
```

Comment:
속성에 익명 함수를 사용한 것이 인상적이네요.
객체 내의 함수를 지정할 때는 속성을 익명 함수로 지정하는 방법도 있지만, "객체"를 사용하는 목적에 따라서 객체 내 메소드 정의를 사용하는 것도 고려해보시면 좋겠습니다.

```suggestion
    save() {
```

이렇게 하면 써야 될 코드도 줄고 보기도 좋아졌죠?

또한, 익명 함수를 속성에 지정하는 것과 객체 내 메소드에는 `this`객체 바인딩에서 차이가 있습니다. [자바스크립트 this 바인딩](https://jeonghwan-kim.github.io/2017/10/22/js-context-binding.html)을 참고해주세요. 객체를 사용하는 목적에 따라서 `this`를 참조할 일이 많다면(여기선 localStorageModel이 되겠죠?) 메소드를 사용하는 것이 좋아요. 익명 함수(`()=> {}`) 내부에서 `this`은 전역 객체(`global`, `window`)를 참조해버리거든요!

Reply:
저도 공부하고 갑니다;; 총총총...

Reply:
정말 생각치도 못헀던 조언이네요!!! 
제가 익명함수를 사용한 이유는, 오직 코드의 가독성 때문이었어요. ㅋㅋㅋㅋ
이름을 앞에 쓰고 콜론을 붙이면 나중에 보더라도 바로 눈에 들어와서 좋았거든요. 또한 제가 일전에 자주 쓰던 방식이었습니다 ㅎㅎ 근데 웹프론트에선 저런 위험한 상황이 있는지 몰랐네요. 감사합니다.

Reply:
운영진 리뷰까지 꼼꼼히 봐주시다니 최고에요😭